### PR TITLE
DRAFT; Spanish/ES translations

### PR DESCRIPTION
**Pull request by Boran**
Describe the changes you have conducted here

Preview the messages on https://fink.inlang.com/github.com/Boran/soaringmeteo_transDE .

### CODE REVIEW

File: frontend/messages/es.json [hunk 1]
```diff
@@ -1,3 +1,186 @@
...
+  "menuOverlay": "Cubrir",
+  "menuOverlayLegend": "Mostrar una superposición en el mapa",
+  "menuWind": "Viento",
+  "menuWindLegend": "Mostrar la fuerza y dirección del viento en un nivel de elevación específico",
+  "menuForecastData": "Datos de pronóstico",
+  "menuModel": "Modelo",
+  "menuGFSLegend": "Pronóstico basado en el modelo GFS",
+  "menuWRFLegend": "Pronóstico  en el modelo WRF",
+  "menuInitializationTime": "Tiempo de inicialización",
+  "menuInitializationTimeLegend": "Hora de inicialización del pronóstico.",
```
Comment (julienrf) [inline frontend/messages/es.json:14]:
```suggestion
  "menuInitializationTimeLegend": "Hora de inicialización del pronóstico",
```

File: frontend/messages/es.json [hunk 2]
```diff
@@ -1,3 +1,186 @@
...
+  "menuSupport": "Soporte Soaringmeteo",
+  "menuDocuments": "Documentos",
+  "menuSoarGFS": "soarGFS (heredado)",
+  "menuSoarWRF": "soarGFS (heredado)",
+  "settingsLanguage": "Idioma",
+  "settingsHelpTranslate": "¡Ayúdanos a traducir!",
+  "settingsWind": "Viento",
+  "settingsWindExplanation": "De forma predeterminada, la velocidad del viento se indica mediante un valor numérico al lado de cada flecha de viento. Si ocultas los valores numéricos, en los meteogramas la velocidad del viento se indicará mediante púas (con una precisión de 2,5 km/h). Consulte la ayuda sobre los meteogramas para obtener más información sobre cómo interpretar las púas.",
+  "settingsWindShowNumericalValues": "Mostrar los valores numéricos",
+  "settingsWindShowNumericalValuesLegend": "Muestra la velocidad del viento como un valor numérico junto a la flecha del viento.",
```
Comment (julienrf) [inline frontend/messages/es.json:27]:
```suggestion
  "settingsWindShowNumericalValuesLegend": "Muestra la velocidad del viento como un valor numérico junto a la flecha del viento",
```

File: frontend/messages/es.json [hunk 3]
```diff
@@ -1,3 +1,186 @@
...
+  "settingsTime": "Tiempo",
+  "settingsTimeExplanation": "La hora se puede mostrar según su zona horaria o en UTC.",
+  "settingsTimeUserTimezone": "La hora usando mi zona horaria actual",
+  "settingsTimeUserTimezoneLegend": "Mostrar la hora usando mi zona horaria actual en lugar de UTC",
+  "settingsTimeUTC": "Mostrar hora según UTC",
+  "settingsTimeUTCLegend": "Mostrar la hora UTC en lugar de usar mi zona horaria actual",
+  "locationSummary": "Resumen",
+  "locationSummaryLegend": "Resumen del pronóstico para esta ubicación.",
```
Comment (julienrf) [inline frontend/messages/es.json:35]:
```suggestion
  "locationSummaryLegend": "Resumen del pronóstico para esta ubicación",
```

File: frontend/messages/es.json [hunk 4]
```diff
@@ -1,3 +1,186 @@
...
+  "locationMeteogram": "Meteograma",
+  "locationMeteogramLegend": "Meteograma para este lugar",
+  "locationSounding": "Sondeo",
+  "locationSoundingLegend": "Sonando para este momento y lugar.",
```
Comment (julienrf) [inline frontend/messages/es.json:39]:
```suggestion
  "locationSoundingLegend": "Sonando para este momento y lugar",
```

File: frontend/messages/es.json [hunk 5]
```diff
@@ -1,3 +1,186 @@
...
+  "period24HoursBefore": "24 horas antes",
+  "periodPrevious": "Período de pronóstico anterior",
+  "periodNext": "Próximo período de pronóstico",
+  "period24HoursAfter": "24 horas después",
+  "layerCloudsAndRain": "Nubes y lluvia",
+  "layerCloudsAndRainLegend": "las nubes y la lluvia",
+  "layerCumulusDepth": "Nubes cúmulos",
+  "layerCumulusDepthLegend": "la profundidad de los cúmulos",
+  "layerSoaringLayerDepth": "Profundidad de la capa límite",
```
Comment (julienrf) [inline frontend/messages/es.json:48]: Would be good to find something shorter :)

File: frontend/messages/es.json [hunk 6]
```diff
@@ -1,3 +1,186 @@
...
+  "zoneEurope": "Europa",
+  "zoneAustralia": "Australia",
+  "zoneHimalaya": "Himalaya",
+  "zoneNorthAmerica": "América del norte",
+  "zoneSouthAfrica": "Sudáfrica",
+  "zoneSouthAmerica": "Sudamerica",
+  "helpIntro1": "Soaringmeteo es un sitio web gratuito de pronóstico del tiempo desarrollado por pilotos apasionados. Por favor considere hacer un",
+  "helpIntro2": "donación",
+  "helpIntro3": "para ayudarnos a cubrir nuestro costo.",
+  "helpCurrentForecast": "Lo que ves es el pronóstico del tiempo para {forecastDate} , del modelo {model} inicializado en {initializationTime} .",
```
Comment (julienrf) [inline frontend/messages/es.json:90]:
```suggestion
  "helpCurrentForecast": "Lo que ves es el pronóstico del tiempo para {forecastDate} , del modelo {model} inicializado en {initializationTime}.",
```

File: frontend/messages/es.json [hunk 7]
```diff
@@ -1,3 +1,186 @@
...
+  "helpMeteogramIsotherm1": "La altitud de la isoterma 0°C se muestra mediante la",
+  "helpMeteogramIsotherm2": "línea negro",
+  "helpMeteogramIsotherm3": "En este ejemplo, comienza alrededor de 3500 m el primer día y aumenta hasta {maxIso} m el tercer día.",
+  "helpMeteogramPressure1": "Por último, la presión atmosférica se muestra mediante la",
+  "helpMeteogramPressure2": "línea roja",
+  "helpMeteogramPressure3": "La escala está a la derecha del diagrama. Aquí oscila alrededor de 1010 hPa.",
+  "helpMeteogramTemperature1": "En la parte inferior vemos la evolución de la temperatura en el suelo.",
```
Comment (julienrf) [inline frontend/messages/es.json:147]:
```suggestion
  "helpMeteogramTemperature1": "En la parte inferior vemos la evolución de la temperatura en el suelo",
```

**Review by julienrf — COMMENTED** [review]
Thank you! In some places the sentences are broken down into several messages and the punctuation should be removed from them. See my suggestions below.

**Review by julienrf — APPROVED** [review]

**Comment by Boran** [comment]
Yes, there is lots to review, this is the initial machine translation with a few changes thatb seemed obvious. Next step is to get a native speaker to review.

**Pull request merged**